.github/checkgroup.yml

```diff
@@ -55,8 +55,10 @@ subprojects:
 
   - id: "pytorch_lightning: Azure GPU"
     paths:
-      - ".azure/gpu-tests.yml"
+      - ".azure/gpu-tests-pytorch.yml"
+      - ".azure/gpu-tests-lite.yml"
```

Comment:
Adding Lite here is not correct as it'd be a separate job.

There needs to be a new `id` for `lightning_lite: Azure GPU`

Reply:
Okay, I made the new group